AdminWebsite/AdminWebsite/Mappers/AddNotificationRequestMapper.cs

```diff
@@ -86,7 +86,13 @@ public static class AddNotificationRequestMapper
                 participant.TelephoneNumber = hearing.GetJudgePhone();
             }
             else if (participant.UserRoleName.Contains("Judicial Office Holder",
-                StringComparison.InvariantCultureIgnoreCase))
+                StringComparison.InvariantCultureIgnoreCase) && hearing.IsJudicialOfficeHolderEJud())
+            {
+                notificationType = NotificationType.HearingAmendmentEJudJoh;
```

Comment:
because the params sent to the template are the same here, you can use a conditional ternary